<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HSV Range Finder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Camera ID:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxCamNo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCamNo">
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStartCapture">
         <property name="text">
          <string>Start Capture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStopCapture">
         <property name="text">
          <string>Stop Capture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBinaryOut">
         <property name="text">
          <string>Binary Output</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="5">
        <widget class="QSpinBox" name="spinBox_S_low">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>S&lt;sub&gt;H&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_S_high</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QSpinBox" name="spinBox_V_high">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="spinBox_H_high">
         <property name="maximum">
          <number>179</number>
         </property>
         <property name="value">
          <number>179</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_H_low">
         <property name="maximum">
          <number>179</number>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QSpinBox" name="spinBox_S_high">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QSpinBox" name="spinBox_V_low">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>H&lt;sub&gt;H&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_H_high</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>H&lt;sub&gt;L&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_H_low</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>V&lt;sub&gt;L&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_V_low</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>S&lt;sub&gt;L&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_S_low</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>V&lt;sub&gt;H&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_V_high</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_S_low">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="horizontalSlider_S_high">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="horizontalSlider_V_high">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_H_high">
         <property name="maximum">
          <number>179</number>
         </property>
         <property name="value">
          <number>179</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="horizontalSlider_V_low">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_H_low">
         <property name="maximum">
          <number>179</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>H&lt;sub&gt;L&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider_H_low</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>H&lt;sub&gt;H&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider_H_high</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>S&lt;sub&gt;L&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider_S_low</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>S&lt;sub&gt;H&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider_S_high</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>V&lt;sub&gt;L&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider_V_low</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>V&lt;sub&gt;H&lt;/sub&gt;</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider_V_high</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_raw_frame">
         <property name="text">
          <string>Raw Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_proc_frame">
         <property name="text">
          <string>Processed Frame</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
